atat_provisioning_wizard_api.yaml

```diff
@@ -16,23 +16,26 @@ paths:
     get:
       tags:
         - portfolioDraft
-      description: Gets all Portfolio Drafts the caller has read access to
+      description: Gets all Portfolio Drafts to which the caller has read access
       operationId: getPortfolioDrafts
       parameters:
         - $ref: '#/components/parameters/offsetParam'
         - $ref: '#/components/parameters/limitParam'
       responses:
         '200':
-          description: Successful operation with empty response body
+          description: Should return up to {limitParam} items
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PortfolioDraftSummary'
-              examples:
-                PortfolioDraftSummaryEx:
-                  $ref: '#/components/examples/PortfolioDraftSummaryEx'
```

Comment:
We still want examples here right?

Reply:
Yes.  I seem to have clobbered them during a rebase.  I've restored the examples.